src/transformStateWithClones.js

```diff
@@ -7,7 +7,38 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let stateCopy = { ...state };
```

Comment:
Avoid reassigning variables, it decreases code [readability](https://blog.sapegin.me/all/avoid-reassigning-variables/)